lib/index.js

```diff
@@ -212,14 +212,31 @@ Server.prototype = {
     }
     this.webApp = makeWebApp(webAppOptions);
     this.webServer.startListening(this.webApp.expressApp);
-    let pluginsLoaded = [];
+    let pluginsLoaded = 0;
+    let pluginCount = 0;
     this.pluginLoader.on('pluginAdded', util.asyncEventListener(event => {
+      pluginCount++;
       return this.pluginLoaded(event.data).then(() => {
-        installLogger.info('Installed plugin: ' + event.data.identifier);
+        pluginsLoaded++;
+        installLogger.info('Installed plugin: ' + event.data.identifier 
+                           + " Successful: (" + pluginsLoaded + "/" + event.count + ")"
+                           + " Attempted: (" + pluginCount + "/" + event.count + ")");
+        if (pluginCount == event.count) {
+          installLogger.info(`Server is ready at ` +
+            `${webAppOptions.serverConfig.node.https.ipAddresses}:${webAppOptions.httpsPort || webAppOptions.httpPort}` +
+            `, Plugins successfully loaded: (${pluginsLoaded}/${event.count})`);
+        }
       }, err => {
         installLogger.warn(`Exception occurred, plugin (${event.data.identifier}) installation skipped. `
-                           +`Message: ${err.message}`);
+                           +`Message: ${err.message}`
+                           + " Successful: (" + pluginsLoaded + "/" + event.count + ")"
+                           + " Attempted: (" + pluginCount + "/" + event.count + ")");
         installLogger.debug(err.stack);
+        if (pluginCount == event.count) {
+          installLogger.info(`Server is ready at ` +
+            `${webAppOptions.serverConfig.node.https.ipAddresses}:${webAppOptions.httpsPort || webAppOptions.httpPort}` +
```

Comment:
This uses https ipAddresses even if https is not used, but uses either https or http port.
To be concise, it would be best to prefer https but if not found, mention http.

Reply:
This also has weird output when there are multiple IP addresses.